### DISCUSSION TITLE
Error in Matlab R2018a when use the functions 'calcDislocationDensities' and 'calcDislocationDensities'.

### DISCUSSION THREAD

**Discussion by pablobruno1** [mtex-toolbox/mtex#604] [Ask Anything] [answered]
Hello everyone,
I hereby would like to ask your help.
I've tried to use both functions 'calcDislocationDensities' and 'calcDislocationDensities' in Matlab R2018a together with Mtex version 5.4 and an error appears for me, for examples by saying: 'Undefined function or variable 'calcDislocationDensities'.
So, does this problem a problem with the Matlab version? or how could I solve this problem?

Best regards,
Pablo Leão

**Comment by kilir** [comment, marked as answer]
Hi,
mtex does not have a function called `calcDislocationDensities`. Maybe you need to explain a little more where you came along this? 
There was a typo in the help for `fitDislocationSystems` but apart from it, I found no occurrence.

Cheers,
Rüdiger

**Reply by pablobruno1** [reply]
Dear Kilir,

Thanks for your reply.
Please look bellow the error message:
![image](https://user-images.githubusercontent.com/75207185/100542908-d563c480-322b-11eb-912c-ca3d811a8705.png)

**Reply by kilir** [reply]
Hi,
the error message tells you that you need to have the Optimization Toolbox which supplies the function `linprog` which is required by `fitDislocationSystems`.
Cheers
Rüdiger

**Reply by pablobruno1** [reply]
Thanks.
